.github/workflows/ci.yml

```diff
@@ -12,7 +12,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [10.x, 11.x, 14.x]
+        node-version: [10.x, 14.x, 16.x]
```

Comment:
TODO: require Node 16 in branch protection rules

Reply:
euh wellek?

Reply:
https://github.com/skedify/sdk-js/settings/branch_protection_rules/1405537

![image](https://user-images.githubusercontent.com/5346497/116524248-74721280-a8d7-11eb-859b-a7d803d4458c.png)

it's not really required, but because web-admin makes use of Node 16, I think we need to enforce all running tests to work on Node 16 too